go/types/opcache.go

```diff
@@ -122,6 +173,45 @@ func (i *opCacheIterator) Op() sequenceItem {
 	return entry
 }
 
-func (i *opCacheIterator) Release() {
+// Insert can be called from any goroutine
+func (opc *LdbOpCache) SetInsert(val Value) {
+	valArray := [initialBufferSize]byte{}
+
+	switch val.Type().Kind() {
+	default:
+		// Set entries are fairly simple to store, simply store the element's hash (or the element itself if bool, number, or string) in the ldb key and the value in ldb value.
+
+		// Note that, if val is a prolly trees, any in-memory child chunks will be written to vrw at this time.
+		ldbKeyArray := [uint32Size + 1 + hash.ByteLen]byte{}
```

Comment:
Similar comment as above, with using a helper method to make the flow of this code here more readable.